src/assets/_project/_blocks/components/forms/recaptcha.js

```diff
@@ -9,51 +9,114 @@
 import keys from '../../data/qg-google-keys';
 
 (function ($, swe) {
-  let googleRecaptchaApiKey =
-    window.location.hostname.search(
-      /\bdev\b|\btest\b|\blocalhost\b|\buat\b/
-    ) !== -1
-      ? keys.defGoogleRecaptcha.uat
-      : keys.defGoogleRecaptcha.prod;
-  let onloadRecaptcha = () => {
-    // eslint-disable-line
-    $('form[data-recaptcha="true"]')
-      .find('input[type="submit"], button[type="submit"]')
-      .on('click', e => {
-        e.preventDefault();
-        let subBtn = e.target;
-        let form = $(subBtn).parents('form');
-        try {
-          grecaptcha.render(subBtn, {
-            sitekey: googleRecaptchaApiKey, //this value will be replaced by build tool. from gulp-config/
-            callback: () => {
-              var response = grecaptcha.getResponse();
-              if (
-                response === '' ||
-                  response === undefined ||
-                  response.length === 0
-              ) {
-                console.log('Invalid recaptcha');
-                return false;
-              } else {
-                form.submit();
-              }
-            },
-          });
-        } catch (e) {
-          grecaptcha.reset();
+  let checkEnv = window.location.hostname.search(/\bdev\b|\btest\b|\blocalhost\b|\buat\b/);
+  let $feedbackForm = $('#qg-page-feedback-form');
```

Comment:
does it need $?

Reply:
This is just a convention to denote a variable is holding the jQuery object. This makes easy for developers to identify and use other jQuery methods on it if there is a need.